cli/bench/main.rs

```diff
@@ -496,6 +502,36 @@ async fn main() -> Result<()> {
     new_data.max_latency = max_latency;
   }
 
+  if benchmarks.contains(&"runtimes") && cfg!(not(target_os = "windows")) {
+    runtimes::setup();
+    let stats = runtimes::ssr()?;
```

Comment:
Can this be run individually like this? 
```
cargo bench --bench deno_bench -- ssr
```

Reply:
I think it would be good if we could run individual benchmarks - that's the design goal of Divy's [earlier work ](https://github.com/denoland/deno/pull/14821)